Drivers/Src/MotorController.c

```diff
@@ -8,13 +8,13 @@
 #define MOTOR_VELOCITY 0x243
```

Comment:
I'm going to assume all of the changes here are for formatting and not read them